content/docs/cucumber/varieties.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cucumber Varieties
+subtitle: "Varieties and Semantic versioning"
+
+weight: 3
+---
+
+Every [Cucumber implementation](/docs/installation) has its own codebase, release cycle and [semver](https://semver.org/) version that is independent from other Cucumber implementations.
+
+The Cucumber team also maintains a list of *varieties* which defines essential characteristics of various Cucumber versions. A Cucumber release may satisfy zero or more characteristics of a variety. If it satisfies all of them it is a member of that variety.
+
+Marking Cucumber releases as part of a variety serves several purposes:
+
+* Define stretch goals for upcoming Cucumber implementations
+* Allow tool vendors to support particular varieties with more ease
+* Documentation becomes easier to read and write as it applies to all releases in the variety
+
+Varieties are named after Wikipedia's [list of Cucumber varieties](https://en.wikipedia.org/wiki/List_of_cucumber_varieties). Higher alphabetical order means more recent.
+
+# Babylon
+
+Characteristics               | [Cucumber-JVM 5.0.0-RC2] | [Cucumber.js master] | [Cucumber-Ruby 4.0.0.rc.2]
+------------------------------|--------------------------|----------------------|--------------------------
+[gherkin] 8.1.x               |                       ✅ |                   ❌ |            ✅ 
+[cucumber-messages] 6.0.x     |                       ✅ |                   ❌ |            ✅ 
+[cucumber-expressions] 6.0.x  |                       ✅ |                   ✅ |            ✅ 
+[tag-expressions] 2.0.x       |                       ✅ |                   ✅ |            ✅ 
+`protobuf` formatter          |                       ✅ |                   ❌ |            ❌ 
+
+## Compatible tools
+
+* [cucumber-html-formatter] 2.x
+* [cucumber-json-formatter] 2.x
+
+# Arola
+
+Characteristics               | [Cucumber-JVM 4.8.0] | [Cucumber.js 6.0.2] | [Cucumber-Ruby 3.1.2]
+------------------------------|----------------------|---------------------|---------------------------
+[gherkin] 5.x                 |                   ✅ |                  ✅ |            ✅ 
+[tag-expressions] 2.0.x       |                   ✅ |                  ✅ |            ❌ (`~> 1.1.0`) 
```

Comment:
given what you have done here with the version available for cucumber ruby (Being lower than the required one). And above I'm guessing not implemented. Should we maybe have some indication it's not implemented yet?